src/state/translations-mixin.ts

```diff
@@ -188,17 +191,8 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
 
     private _applyDirection(hass: HomeAssistant) {
       const direction = computeRTLDirection(hass);
-      this.style.direction = direction;
       document.dir = direction;
```

Comment:
Move this to the helper function too.

Use `computeRTL(hass)` so we can use the same function in both mixins.